packages/mantine/src/components/browser-preview/BrowserPreview.tsx

```diff
@@ -0,0 +1,70 @@
+import {InfoSize16Px} from '@coveord/plasma-react-icons';
+import {Box, ColorSwatch, Flex, Group, Stack, Text, Title, Tooltip, useMantineTheme} from '@mantine/core';
+import {PropsWithChildren} from 'react';
+
+export interface BrowserPreviewProps {
+    /**
+     * Text that will be displayed in the tooltip on the info icon
+     *
+     * @default The final look in your search page may differ due to the customization you made in your page.
```

Comment:
I think there should not be a default, since it will be hard to to track where the string comes from later. We don't have localization of strings on the plasma side and strings don't get reviewed by the doc team here.

Reply:
Yes I agree. I removed the default but I didn't want the tooltip to show with no text if none is specified so I made sure to hide the tooltip completely is such cases. Or we could make this prop required, but I preferred having more flexibility.